dask/dataframe/io.py

```diff
@@ -14,6 +14,7 @@
 import pandas as pd
 import numpy as np
 import dask
+import bcolz
```

Comment:
But this one will need to be guarded.  Ideally we can just import it inside of the to_bcolz function